vendor/engines/secure_rooms/config/locales/en.yml

```diff
@@ -21,8 +21,8 @@ en:
   order_details:
     notices:
       missing_entry:
-        badge: Missing Entry
-        alert: This order's occupancy does not have an entry time. Please ensure that all times are set and there is a price policy for the date this order was fulfilled.
+        badge: Inferred Account
```

Comment:
I think I'd be fine leaving the badge the same. Could be convinced otherwise, though.